qpc_tools/server/ansible/install/roles/server_podman/tasks/start_quipucords_container.yml

```diff
@@ -53,6 +53,5 @@
 
 # With podman we have to set the host to localhost instead of qpc-db, because podman does not create a proxy
 - name: Start Quipucords server Podman container linked to PostgreSQL container
-  shell: "podman run --name {{ server_name }} -p {{ server_port }}:443 -d -e USE_SUPERVISORD={{ use_supervisord }} -e QPC_SERVER_TIMEOUT={{ server_http_timeout }} -e QPC_DBMS_USER={{ db_user }} -e QPC_DBMS_PASSWORD='{{ db_password }}' -e ANSIBLE_LOG_LEVEL={{ ansible_log_level }} -e NETWORK_CONNECT_JOB_TIMEOUT={{ connect_job_timeout }} -e NETWORK_INSPECT_JOB_TIMEOUT={{ inspect_job_timeout }} -e QPC_SERVER_USERNAME={{ server_username }} -e QPC_SERVER_USER_EMAIL={{ server_user_email }} -e QPC_SERVER_PASSWORD='{{ server_password }}' -v {{ server_sshkeys_volume }}:/sshkeys{{ dir_mount_selinux }} -v {{ server_data_volume }}:/var/data{{ dir_mount_selinux }} -v {{ server_log_volume }}:/var/log{{ dir_mount_selinux }} -i {{ server_image_and_tag }}"
-  become: true
-  ignore_errors: yes
\ No newline at end of file
+  shell: "podman run --name {{ server_name }} -p {{ server_port }}:443 -d -e USE_SUPERVISORD={{ use_supervisord }} -e QPC_SERVER_TIMEOUT={{ server_http_timeout }} -e QPC_DBMS_USER={{ db_user }} -e QPC_DBMS_PASSWORD='{{ db_password }}' -e QPC_DBMS_HOST=$(podman inspect -f '{{ '{{' }} .NetworkSettings.IPAddress {{ '}}' }}' {{ db_name }}) -e ANSIBLE_LOG_LEVEL={{ ansible_log_level }} -e NETWORK_CONNECT_JOB_TIMEOUT={{ connect_job_timeout }} -e NETWORK_INSPECT_JOB_TIMEOUT={{ inspect_job_timeout }} -e QPC_SERVER_USERNAME={{ server_username }} -e QPC_SERVER_USER_EMAIL={{ server_user_email }} -e QPC_SERVER_PASSWORD='{{ server_password }}' -v {{ server_sshkeys_volume }}:/sshkeys{{ dir_mount_selinux }} -v {{ server_data_volume }}:/var/data{{ dir_mount_selinux }} -v {{ server_log_volume }}:/var/log{{ dir_mount_selinux }} -i {{ server_image_and_tag }}"
```

Comment:
@gtanzillo I replaced the `--network` param on postgres container with an env here `-e QPC_DBMS_HOST=$(podman inspect -f '{{ '{{' }} .NetworkSettings.IPAddress {{ '}}' }}' {{ db_name }})`